lib/packs/private/interactive_cli/pack_directory_selector.rb

```diff
@@ -0,0 +1,28 @@
+# typed: strict
+
+module Packs
+  module Private
+    module InteractiveCli
+      class PackDirectorySelector
+        extend T::Sig
+
+        sig { params(prompt: TTY::Prompt, question_text: String).returns(String) }
+        def self.select(prompt, question_text: 'Select a directory')
+          directories = []
+
+          Packs::Specification.config.pack_paths.each do |path|
+            directories << Dir.glob(path).select { |f| File.directory? f }
+          end
```

Comment:
Is there a way we can test this?

Reply:
Yes - good call. I added a test, and it caught that `directories` needed to be flattened 😅